data-labeling/environments/data-labeling/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/data-labeling
```

Comment:
Have you created a PR in the mcr repo for this image?

Reply:
Created a PR in the mcr repo https://github.com/microsoft/mcr/pull/2529

Reply:
Just merged the pr